File: Data Structures & Algorithms/two-integer-sum/submission-0.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:

        #Time Complexity O(n^2)
        
        for i in range(len(nums)-1):
            num = target - nums[i]
            for j in range(i+1,len(nums)):
                if(nums[j] == num):
                    return [i,j]

        

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int n = nums.length;
        int i = 0, j = 0;
        while ( j < n ){
            nums[i] = nums[j];
            while (j < n && nums[j] == nums[i]){
                j++;
            }
            i++;
        }
        return i;
    }
}